src/components/Layout.tsx

```diff
@@ -7,11 +7,11 @@ const { Content } = AntLayout;
 import Navbar from "./Navbar";
 import "../style/index.sass";
 import useSiteMetadata from "./SiteMetadata";
-import { container } from "../style/layout.module.css";
+const { container } = require("../style/layout.module.css");
 
 const OFF_WHITE = "#F2F2F2";
 const DARK_BLUE = "#00215F";
-const WHITE = "#FFFFFF";
+export const WHITE = "#FFFFFF";
```

Comment:
not sure if there is a shared constants file but this seems like a good candidate for it, if it already exists.  Would be weird to depend on Layout.tsx just to get WHITE defined.